### ISSUE TITLE
Colour Contrast/Use of Color issues with Bootstrap markup

### ISSUE THREAD

**Issue by mmafe** [ccswbs/hjckrrh#676]
Found the following styling issues when reviewing how our theme colours look when using Bootstrap markup.

**Link Colour over Table Rows with Contextual Background Colours**
1. Link colour (#2174BB) on Table Contextual "Success" Row (#DFF0D8) has 4.11 ratio - Fails on SC 1.4.3 Contrast (Minimum)
2. Link colour (#2174BB) on Table Contextual "Danger" Row (#F2DEDE) has 3.8 ratio - Fails on SC 1.4.3 Contrast (Minimum)
![image](https://cloud.githubusercontent.com/assets/1927902/24508211/c5a0d31a-1530-11e7-9274-7e690490585c.png)

3. Recommend adding an underline to links in alerts. This is a pre-existing issue, but a quick fix. Currently there's no way to tell the difference between bold and linked text in alerts. This is a failure on SC 1.4.1 Use of Color (A).
![image](https://cloud.githubusercontent.com/assets/1927902/24510594/adaf4b90-1537-11e7-9901-4522d638c423.png)

4. Recommend adding an underline to links in panel titles. Similar to alerts, there's no way to tell when there's linked text in an alert. This is a failure on SC 1.4.1 Use of Color (A).
![image](https://cloud.githubusercontent.com/assets/1927902/24512384/6a0c561c-153c-11e7-92d4-5e93b60336bf.png)

5. Active List Group paragraph text is being overridden and appears as dark text colour over dark blue (see ug2015-v1.css) - recommend ensuring the text colour works on the dark background, as well as if a link was included (eg. underlines might be needed if we use the lighter colour that typically would be used). This would be a failure on SC 1.4.3. Contrast (Minimum) (AA) and if we do not use an underline with the lighter colour, potentially SC 1.4.1 Use of Colour (A) 
![image](https://cloud.githubusercontent.com/assets/1927902/24511067/f5a7c3fe-1538-11e7-90eb-1b9de54e6d84.png)

6. Default panel (panel-default) headings (panel-heading) are set to #333333. If a link is included in the panel heading, the ratio between #333333 and #2174BB is not sufficient to meet the 3:1 ratio for SC 1.4.1 Use of Color (A)
![image](https://cloud.githubusercontent.com/assets/1927902/24511921/17572506-153b-11e7-8067-c24ed21f771d.png)
